### PR TITLE
feat(discv2): toggle detail view

### PR DESCRIPTION
**Pull request by doralchan**
Screen Shot 2019-12-05 at 1 50 10 PM
Screen Shot 2019-12-05 at 2 37 34 PM

- Added  a hide/show button to the event detail 
- Added an "OR" to the landing page

### CODE REVIEW

**Comment by dashed** [comment]
@doralchan looks like you need to rebase against `master` to resolve some conflicts.

**Comment by doralchan** [comment]
Yep - did that already but there are still issues with the undefined event and graph 😭

File: src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx [hunk 1]
```diff
@@ -117,6 +130,9 @@ class EventDetailsContent extends AsyncComponent<Props, State & AsyncComponent['
           />
           <EventHeader event={event} />
           <Controller>
+            <StyledButton size="small" onClick={this.toggleSidebar}>
+              {isSidebarVisible ? 'Hide Details' : 'Show Details'}
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx]:
```suggestion
              {isSidebarVisible ? t('Hide Details') : t('Show Details')}
```

File: src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx [hunk 2]
```diff
@@ -146,8 +162,8 @@ class EventDetailsContent extends AsyncComponent<Props, State & AsyncComponent['
               projectId={this.projectId}
               eventView={eventView}
             />
-          </Main>
-          <Side>
+          </div>
+          <div style={{gridColumn: '2/3', display: isSidebarVisible ? '' : 'none'}}>
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx]: Could we use a styled component for this. Inline styles get unwieldy quick.

**Comment by dashed** [comment]
@doralchan the ts fix commit seems to have been lost during the rebase. I've added one in https://github.com/getsentry/sentry/pull/15962/commits/0c47a13c5bc466528d83b1a08ec56d749c84febf

**Review by markstory — APPROVED** [review]
Looks good to me.

File: src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx (hunk 1 above)
Comment (doralchan) [reply src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx]: Crap I just saw this - will do it in a new branch

File: src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx (hunk 2 above)
Comment (doralchan) [reply src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx]: Yeah - I tried to do it in a styled component but it wasn't working. Will try this in the new branch.

**Pull request merged**